buda/entities/signup.proto

```diff
@@ -18,8 +18,8 @@ message Signup {
 
     enum SignupClient {
         WEB = 0;
-        PUBLISH = 1;
-        ANALYZE = 3;
+        // PUBLISH = 1; --deprecated value
```

Comment:
@mallen5311 Since I don't think we actually ever used these values, I would say we could just delete these lines instead of commenting them out.

Reply:
Ah, ok, good advice.  I'll make that change now.